index.js

```diff
@@ -2295,25 +2320,68 @@
     impl: isRight
   };
 
-  //# fromEither :: b -> Either a b -> b
+  //# fromLeft :: a -> Either a b -> a
+  //.
+  //. Takes a default value and an Either, and returns the Left value
+  //. if the Either is a Left; the default value otherwise.
+  //.
+  //. See also [`fromRight`](#fromRight).
+  //.
+  //. ```javascript
+  //. > S.fromLeft (0) (S.Left (42))
+  //. 42
+  //.
+  //. > S.fromLeft (0) (S.Right (42))
+  //. 0
+  //. ```
+  function fromLeft(x) {
+    return either (I) (K (x));
+  }
+  _.fromLeft = {
+    consts: {},
+    types: [a, $.Either (a) (b), a],
+    impl: fromLeft
+  };
+
+  //# fromRight :: b -> Either a b -> b
   //.
   //. Takes a default value and an Either, and returns the Right value
   //. if the Either is a Right; the default value otherwise.
   //.
+  //. See also [`fromLeft`](#fromLeft).
+  //.
   //. ```javascript
-  //. > S.fromEither (0) (S.Right (42))
+  //. > S.fromRight (0) (S.Right (42))
   //. 42
   //.
-  //. > S.fromEither (0) (S.Left (42))
+  //. > S.fromRight (0) (S.Left (42))
   //. 0
   //. ```
-  function fromEither(x) {
+  function fromRight(x) {
     return either (K (x)) (I);
   }
-  _.fromEither = {
+  _.fromRight = {
     consts: {},
     types: [b, $.Either (a) (b), b],
-    impl: fromEither
+    impl: fromRight
```

Comment:
Should use `B (C (either) (I)) (K)` here to match [`fromLeft`](https://github.com/sanctuary-js/sanctuary/pull/644/files#r309266193)?

Reply:
@davidchambers :wave:

Reply:
I find the existing implementation of `fromRight` easy to read. Let's keep it. Let's also keep your implementation of `fromLeft`, as there is value in similar things looking similar. :)